cocotb/share/makefiles/Makefile.deprecations

```diff
@@ -2,6 +2,10 @@
 # Licensed under the Revised BSD License, see LICENSE for details.
 # SPDX-License-Identifier: BSD-3-Clause
 
+ifdef COCOTB_NVC_TRACE
+    $(warning COCOTB_NVC_TRACE is deprecated, see the "Simulator Support" section in the documentation.)
+endif
+
```

Comment:
Since this PR is not a 100% revert anyway, we don't need to reintroduce this IMO.

```suggestion
```